test/test_example.py

```diff
@@ -12,6 +12,8 @@
 
 @genty
 class ExampleTests(TestCase):
+    _multiprocess_can_split_ = True
```

Comment:
I have it so that when manually testing... I can tickle the scenario. Without the *fix* the tests will fail. And with the *fix* the tests pass. So I think this should be committed. I can add a comment.